nparse.py

```diff
@@ -92,14 +93,23 @@ def _toggle(self):
             if self._log_reader:
                 self._log_reader.deleteLater()
                 self._log_reader = None
+            location_service.stop_location_service()
+            for parser in self._parsers:
+                try:
+                    parser.shutdown()
+                except:
+                    print("Failed to shutdown parser: %s" % parser.name)
             self._toggled = False
 
     def _parse(self, new_line):
         if new_line:
             timestamp, text = new_line  # (datetime, text)
             #  don't send parse to non toggled items, except maps.  always parse maps
-            for parser in [parser for parser in self._parsers if config.data[parser.name]['toggled'] or parser.name == 'maps']:
-                parser.parse(timestamp, text)
+            for parser in [parser for parser in self._parsers]:
```

Comment:
So, this is technically an efficiency loss, but I don't think the performance hit is significant enough to be noticed... This is necessary for the windows to handle toggling themselves via log commands.